webapp/javascript/components/TimelineChart/TimelineChartPlugin.ts

```diff
@@ -57,7 +58,17 @@ import { getUTCdate } from '@webapp/util/formatDate';
       }
     }
 
+    const setHoverCursor = (isHover: boolean) => {
+      if (this.isHover !== isHover) {
```

Comment:
```suggestion
      const hasChanged = this.isHover !== isHover;
      if (hasChanged) {
```

a nit, but I found these are clearer to read, any chance to reduce mental overhead is a good idea